ds_judgements_public_ui/static/markdown/court_descriptions/default.md

```diff
@@ -0,0 +1 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent non viverra dui, quis dignissim metus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. In a risus in tellus suscipit porta. Curabitur risus erat, congue vel lectus non, pellentesque aliquam nulla. Morbi purus lectus, consequat pharetra metus hendrerit, convallis varius neque. Nullam vitae sapien purus. Aenean fermentum quam et nisl pharetra, et gravida dui posuere. In aliquet eros a lacus congue pellentesque eu in risus. Donec non venenatis metus.
```

Comment:
Not loving lorem ipsum text here: an empty document might make more sense.

Reply:
yeah  I think this should go - i had that there for testing but if we don't yet have a generic text, best to just show nothing